clang/lib/Sema/SemaTemplateInstantiate.cpp

```diff
@@ -805,7 +805,6 @@ Sema::InstantiatingTemplate::InstantiatingTemplate(
           SemaRef, CodeSynthesisContext::BuildingDeductionGuides,
           PointOfInstantiation, InstantiationRange, Entity) {}
 
-
```

Comment:
Unrelated

Reply:
Sorry, `git clang-format` keeps modifying this.